optimum/exporters/neuron/base.py

```diff
@@ -237,3 +238,31 @@ def generate_dummy_inputs(self, return_tuple=True, **kwargs) -> Union[Dict[str,
             return tuple(dummy_inputs.values())
         else:
             return dummy_inputs
+
+    def check_model_inputs_order(
+        self,
+        model: PreTrainedModel,
+        dummy_inputs: Dict[str, torch.Tensor],
```

Comment:
```suggestion
        dummy_inputs: Dict[str, "torch.Tensor"],
```

Reply:
`torch` is imported so imo there is no need to use quotes here.

Reply:
Then you should remove present somewhere else in the code!

Reply:
Yeah, quotes for `torch.Tensor` were removed here [36f2dbd](https://github.com/huggingface/optimum-neuron/pull/33/commits/36f2dbd804db355459c8665b2cd4733e44d41295). My bad, I was not careful enough on typing and docstrings in the exporter PR.